manager/README.md

```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
+Disk /dev/sda: 11.0GB
+Sector size (logical/physical): 512B/512B
+Partition Table: gpt
+Disk Flags:
+
+Number  Start   End     Size    File system  Name  Flags
+        17.4kB  1049kB  1031kB  Free Space
+14      1049kB  5243kB  4194kB                     bios_grub
+15      5243kB  116MB   111MB   fat32              boot, esp
+ 1      116MB   2361MB  2245MB  ext4
+        2361MB  3435MB  1074MB  Free Space
+```
+
+Partition 1 contains the root file system. To resize this partition type:
+```
+resizepart 1
+```
+The parted command will then ask you the following question:
+```
+Warning: Partition /dev/sda1 is being used. Are you sure you want to continue?
+Yes/No?
+```
+Type `yes`.
+
+Next, when asked about the root partition's new end number, enter the free space's end number. In our example, this is the number 11GB. Example:
+```
+End?  [2361MB]? 3435MB
+```
+Exit the parted command by typing `quit`.
+
+The last step is to resize the filesystem of the root file system partition. To resize it, execute:
+```sh
+resize2fs /dev/sda1
+```
+Check the new size of the partition:
+```sh
+df -h
+# Output: /dev/sda1       3.0G  1.5G  1.5G  50% /
+```
+
+#### Set up the network interface
+
+We need to set up the network interface in order to be establish an HTTP and gRPC communication of the VM and the "external world".
+
+We are still in the VM. If the VM is not started, start it with `./start_VM` and log in with root. Set up the network interface:
+
+```sh
+# Find out the virtual network interface and store it in NETWORK_INTERFACE env var.
+NETWORK_INTERFACE=$(ip addr | awk '/^2: /{sub(/:/, "", $2); print $2}')
+# To be sure, you can compare the output of the following command with the output of 'ip addr' command.
+echo $NETWORK_INTERFACE
+
+# Bring the specified network interface up.
+ip link set dev $NETWORK_INTERFACE up
+# Use dhclient to automatically obtain an IP address and configure the specified network interface.
+dhclient $NETWORK_INTERFACE
+```
+
+#### Install Go language
+
+We install go language in our VM in order to compile an `Agent` in the VM.
+
+```sh
+# Download the Go programming language distribution version 1.21.0 for Linux AMD64.
+wget https://go.dev/dl/go1.21.4.linux-amd64.tar.gz
+
+# Remove any existing Go installation in /usr/local/go and extract the downloaded Go archive there.
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.21.4.linux-amd64.tar.gz
+
+# Update the PATH environment variable to include the directory containing the Go binary.
+export PATH=$PATH:/usr/local/go/bin
+
+# Verify the installed Go version by running the "go version" command.
+go version
+# => # go version go1.21.4 linux/amd64
+```
+
+#### Build and run Agent
+
+```sh
+git clone https://github.com/ultravioletrs/cocos
+cd cocos
+
+# Build the 'agent' executable and save it to the 'build' directory.
+go build -o build/agent cmd/agent/main.go
+
+# Start the 'agent' executable in the background using '&' at the end.
+./build/agent &
```

Comment:
After you have added the NATS section, can please explain how to start the `agent` with NATS server?